docs/getting-started.rst

```diff
@@ -17,10 +16,226 @@ Creating a New Project
 Adding a New App
 ~~~~~~~~~~~~~~~~
 
+Defining models
+~~~~~~~~~~~~~~~~~~~~~~~
+
+  .. code-block:: python
+
+    #quickstart/models.py
+    from django.db import models
 
-Defining a gRPC Service
+    class User(models.Model):
+    full_name = models.CharField(max_length=70)
+
+        def __str__(self):
+            return self.full_name
+    
+    class Post(models.Model):
+        pub_date = models.DateField()
+        headline = models.CharField(max_length=200)
+        content = models.TextField()
+        user = models.ForeignKey(User, on_delete=models.CASCADE)
+    
+        def __str__(self):
+            return self.headline 
+    
+    class Comment(models.Model):
+        pub_date = models.DateField()
+        content = models.TextField()
+        user = models.ForeignKey(User, on_delete=models.CASCADE)
+        post = models.ForeignKey(Post, on_delete=models.CASCADE)
+    
+        def __str__(self):
+            return self.pub_date 
+
+
+Defining serializers
 ~~~~~~~~~~~~~~~~~~~~~~~
+In this example, our serializers inherit from ModelProtoSerializer, which is simply an inheritance of DRF's ModelSerializer.
+For more extensive use, you can use all the DRF serializer methods: `Django REST framework serializers <https://www.django-rest-framework.org/api-guide/serializers/>`_.
+  .. code-block:: python
+
+    #quickstart/serializers.py
+    from django_socio_grpc import proto_serializers
+    from rest_framework import serializers
+    from quickstart.models import User, Post, Comment
+
+    # It is a preferable option to import the pb2 response in this way because,
+    # for the moment, django-socio-grpc does not dynamically detect the correct response
+    # from the pb2 files to associate with the serializer.
+    # If a response does not exist, it will raise an exception, and block the generation of proto files.
+    # Catching the exception helps to solve this problem,
+    # and allows django-socio-grpc to generate proto files normally.
+    try:
+        from quickstart.grpc.quickstart_pb2 import (
+            UserResponse,
+            UserListResponse,
+            PostResponse,
+            PostListResponse,
+            CommentResponse,
+            CommentListResponse
+        )
+        except ImportError:
+            UserResponse = None
+            UserListResponse = None
+            PostResponse = None
+            PostListResponse = None
+            CommentResponse = None
+            CommentListResponse = None
+
```

Comment:
These do not exist yet in this quickstart, they come after the proto generation

Reply:
Done